app/controllers/user_controller.rb

```diff
@@ -1,27 +0,0 @@
-class UserController < ApplicationController
```

Comment:
@andylouisqin  Yea, we have a users_controller, so the user_controller was not necessary